doc/api/next_api_changes/deprecations/19019-AL.rst

```diff
@@ -0,0 +1,5 @@
+*resize_callback* parameter to ``FigureCanvasTk``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+This parameter was never used internally and is deprecated.  Tk-level custom
+event handlers for resize events can be added to a ``FigureCanvasTk`` using
+e.g. `get_tk_widget().bind('<Configure>', ..., True)`.
```

Comment:
```suggestion
e.g. ``get_tk_widget().bind('<Configure>', ..., True)``.
```

(Doc build failure: `/home/circleci/project/doc/api/next_api_changes/deprecations/19019-AL.rst:3: WARNING: py:obj reference target not found: get_tk_widget().bind('<Configure>', ..., True)`)

Reply:
yup